src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -189,6 +189,13 @@
       "You cannot remember other memories until you have #yClarity of your current memory."
     ]
   },
+  "stsjorbsmod:OldBookPower": {
+    "NAME": "Old Book",
+    "DESCRIPTIONS": [
+      "When you would die, instead #yDestroy Old Book.",
+      "When you would die, instead #yDestroy Old Book+ and heal #g%1d percent of your max HP."
```

Comment:
It might be a good idea for card references to have a color as well. Not sure if we did anything with Wrath Memory.